docs/embedded.md

```diff
@@ -0,0 +1,87 @@
+# Embedded C#/WinRT Support
+
+## Overview
+
+Embedded support is a C#/WinRT feature that allows you to have the WinRT layer of your projection compiled into the projection itself. This is done by setting the build property `CsWinRTEmbedded` to `true` in the app or library's project file. 
+
+Enabling embedded support brings in the WinRT.Runtime sources to the project and automatically sets them to be compiled with the project.
+Any self-contained usage of Windows SDK types must be specified using the property `CsWinRTIncludes`. The specified types will be projected and embedded into the project's *.dll*. This means projects using embedded support no longer have a dependency on `Microsoft.Windows.SDK.NET.dll` and `WinRT.Runtime.dll`.
+
+For an example, you can look at the [sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/TestEmbedded). 
+
+## Scenarios
+
+This feature allows C# apps and libraries to target `net5.0` (and above), .NET Framework 4.6.1+,`netcoreapp3.1`, and `netstandard2.0` while also using the Windows SDK.
+Moreover, a library can target `netstandard2.0` and be able to run on NetFX, Net Core and Net 5. 
+
+## Usage 
+
+Using embedded support allows you to target non-Windows TFMs and older versions of .NET. 
+You will need to set a few properties:
+  * `CsWinRTEmbedded` - must be set to `true` to include the sources
+  * `CsWinRTWindowsMetadata` - must be set to the Windows SDK release to use for Windows APIs
+  * `LangVersion` - must be set to `9` because the WinRT.Runtime sources use C# 9
+

```

Comment:
Maybe also add `CsWinRTIncludes` here as a required property, and note that an error (CS0246) will highlight any missing namespaces